src/config.cr

```diff
@@ -37,7 +37,8 @@ struct Twiliosim::Config
     end
   end
 
-  protected def self.float_env(var_name) Float64 | Nil
+  protected def self.float_env(var_name)
```

Comment:
you probably wanted 

`protected def self.float_env(var_name) : Float64 | Nil`

Reply:
Thanks!!!